scripts/deploy-backend.sh

```diff
@@ -97,18 +97,22 @@ generate_jwt_cert()
 }
 
 update_kibana()
-{  
+{
   # Add network policy allowing Kibana to talk to the proxy and to allow the backend to talk to Kibana
   cf add-network-policy "$CGAPPNAME_BACKEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
   cf add-network-policy "$CGAPPNAME_FRONTEND" "$CGAPPNAME_KIBANA" --protocol tcp --port 5601
   cf add-network-policy "$CGAPPNAME_KIBANA" "$CGAPPNAME_FRONTEND" --protocol tcp --port 80
+
+  # Upload dashboards to Kibana
+  CMD="curl -X POST $CGAPPNAME_KIBANA.apps.internal:5601/api/saved_objects/_import -H 'kbn-xsrf: true' --form file=@/home/vcap/app/tdpservice/search_indexes/kibana_saved_objs.ndjson"
+  cf run-task $CGAPPNAME_BACKEND --command "$CMD" --name kibana-obj-upload
```

Comment:
Could we run into a timeout/hang here based on what we've seen with previous use of run-task? The outcome would just be a functional but empty kibana, yes?

Reply:
@elipe17 🆙 ?

Reply:
To answer Andrews question; Since we don't specify that we need to wait for the `run-task` to finish/succeed, it would be the case that we don't get hung up here. Kibana may not necessarily be empty though if this task failed. If it has succeeded once before then Kibana will have that previous dashboard state still. It just might not have the latest dashboard updates if there happen to be any when this command is running.